Automattic/jetpack
# Add Hosting Provider guess

Fixes https://github.com/Automattic/jetpack-roadmap/issues/1039

## Proposed changes:
* Add hosting provider guess using nameservers (from wpcom)
* Implement the initial state value indicating whether user is guessed as coming from hosting provider or not

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Run the site on JN / Ephemeral Site
* Activate Jetpack Beta plugin with this branch, and go to `/wp-admin/admin.php?page=my-jetpack`
* Check that `window.myJetpackInitialState.isUserFromHostPartner` exists
* (currently, there is no way to check whether hosting provider is guess properly)
